labs.py

```diff
@@ -0,0 +1,3 @@
+from demos.Agent_Based_Modeling.prey_predator_abm.model.config import exp
+
+model_dir = 'demos'
```

Comment:
D. needed for labs
**MUST** have `labs.py` in project root directory
**AND** import the `Experiment()` used for the simulation
**AND** specify the directory you model(s) are written within using `model_dir`